testgrid/config/config_test.go

```diff
@@ -174,6 +174,9 @@ func TestConfig(t *testing.T) {
 
 	sqJobs := strings.Split(sqData.Data["submit-queue.jenkins-jobs"], ",")
 	for _, sqJob := range sqJobs {
+		if sqJob == "\"\"" { // ignore empty list of jobs
```

Comment:
I'm following the pattern used for other empty values like `submit-queue.required-contexts`.

https://github.com/kubernetes/test-infra/pull/2564/files#diff-8c98b46a995621a05311c6330e8926c3R33

I suspect there's some goofy bash reason why we want to do it this way.